snippets/auth/index.py

```diff
@@ -634,6 +635,418 @@ def send_custom_email(email, link):
     del email
     del link
 
+def create_saml_provider_config():
+	# [START create_saml_provider]
+    saml = auth.create_saml_provider_config(
+        display_name='SAML provider name',
+        enabled=True,
+        provider_id='saml.myProvider',
+        idp_entity_id='IDP_ENTITY_ID',
+        sso_url='https://example.com/saml/sso/1234/',
+        x509_certificates=[
+            '-----BEGIN CERTIFICATE-----\nCERT1...\n-----END CERTIFICATE-----',
+            '-----BEGIN CERTIFICATE-----\nCERT2...\n-----END CERTIFICATE-----',
+        ],
+        rp_entity_id='P_ENTITY_ID',
+        callback_url='https://project-id.firebaseapp.com/__/auth/handler')
+
+    print('Created new SAML provider:', saml.provider_id)
+	# [END create_saml_provider]
+
+def update_saml_provider_config():
+	# [START update_saml_provider]
```

Comment:
Do we put the `[START]` `[END]` tags inside the function or outside?
I have seen them outside in some code samples

Reply:
We normally put them inside, since the custom function signatures do not add any useful information to the snippet.